api/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Alpine Linux doesn't come with bash installed, so wait-for-it.sh dies immediately with `env: can't execute 'bash': No such file or directory`. wait-for-it uses statement which require bash, so I recommend switching to one of the sh-compatible options from https://docs.docker.com/compose/startup-order/ like https://github.com/Eficode/wait-for.

Reply:
I did not get the error yesterday, but get it today :) thanks Justin
updated